README.md

```diff
@@ -183,6 +190,26 @@ You can also install tyssue from PyPi, this is a CGAL-less version (pure python)
 
 See [INSTALL.md](INSTALL.md) for a step by step install, including the necessary python environment.
 
+## How to cite 
+If you find `tyssue` useful please cite [this repository](https://github.com/DamCB/tyssue) using its DOI as follows:
+
+> Theis, Suzanne, Gay, (2021). Tyssue: an epithelium simulation library. Journal of Open Source Software, 6(62), 2973 doi:[https://doi.org/10.21105/joss.02973](https://doi.org/10.21105/joss.02973)
+> 
+> Zenodo doi:10.5281/zenodo.4817609
+
+## Publications
+If `tyssue` has made a substantial contribution to your work, please edit this file and open a PR.
+
+[rahman_cell_2022] Rahman T and Wan LQ. Cell Jamming Regulates Epithelial Chiral Morphogenesis. SSRN Scholarly Paper. March 2022. [https://doi.org/10.2139/ssrn.4052248](https://doi.org/10.2139/ssrn.4052248)
+
```

Comment:
Also:


George Courcoubetis, Chi Xu, Sergey V. Nuzhdin, Stephan Haas Avalanches during epithelial tissue growth; Uniform Growth and a drosophila eye disc model, PLOS Computational Biology
https://doi.org/10.1371/journal.pcbi.1009952